sonarcloud/create_sonarcloud_projects-token.sh

```diff
@@ -67,32 +92,40 @@ curl_post() {
 curl_get() {
   local api_path="$1"
   local api_opts="$2"
+  local dry="${3:-${DRY_RUN}}"
 
-  if ! ${DRY_RUN}; then
+  if ! ${dry}; then
```

Comment:
Is there a good reason to use an extra local variable instead of `${DRY_RUN}` directly?

Reply:
Ok. To be honest, the `curl_get()` does not really need a DRY_RUN mode at all. It's just a curl GET call.

Reply:
done